src/data/pages/tos.de.md

```diff
@@ -0,0 +1,229 @@
+
+Server-Log-Files werden maximal 5 Tage aufbewahrt.
+
+#### Verwendung von Cookies
+
+Unsere Webseite verwendet Cookies. Bei Cookies handelt es sich um Textdateien, die mit Hilfe des Browsers auf dem Betriebssystem deines Gerätes abgelegt werden, wenn du unsere Webseite aufrufst. Cookies richten auf deinem Rechner keinen Schaden an und enthalten keine Viren.
+
+Wir verwenden ausschliesslich sogenannte “Session-Cookies”. Diese werden bei einem Logout oder nach 12 Stunden automatisch gelöscht. Sie sind ausserdem zwingend notwendig für die Login-Funktion, du hast daher keine Möglichkeit dagegen Widerspruch zu erheben. Die Cookies werden aber auf deinem Rechner gespeichert. Du hast daher die volle Kontrolle und kannst diese vollständig löschen oder durch Änderung der Einstellungen in deinem Browser die Übertragung einschränken. Falls du Cookies für _eCamp_ deaktivierst, wirst du dich nicht mehr einloggen können und somit die Applikation nicht mehr nutzen können.
+
+#### Registrierung eines Nutzenden-Kontos
+
+Wir unterscheiden zwei Stufen von Nutzenden:
+
+1. Ohne Registrierung: Du kannst frei und anonym auf die rein informative Startseite [https://ecamp3.ch](/de/) zugreifen und dich auf ihr bewegen.
+2. Um die eigentlichen Lagerplanungs-Funktionen unserer Applikation zu nutzen, ist eine Registrierung erforderlich. Dazu muss ein Nutzenden-Konto erstellt werden mit einem zweiteiligen Namen, einer gültigen E-Mail-Adresse sowie einem Passwort. Die Registrierung beinhaltet einen Double-Opt-In, sprich während der Registrierung musst du zwingend die Nutzungsbedingungen und Datenschutzerklärung akzeptieren und du erhältst nach Ausfüllen des Registrierungsformulars zwecks Verifizierung eine Bestätigungs-E-Mail. Erst danach wird dein Nutzenden-Konto aktiviert. Von da an kannst du dein Nutzenden-Profil ergänzen oder auch das Passwort bei Bedarf ändern.
+
+#### Einsicht in deine Daten
+
+Sämtliche persönlichen Daten, die wir über dich dauerhaft speichern, kannst du jederzeit in deinem Nutzenden-Profil einsehen.
+
+#### Löschen deines Nutzenden-Kontos
+
+Möchtest du dein Konto nicht mehr weiterführen, so kannst du uns [kontaktieren](/de/kontakt) um eine Löschung zu beantragen. Dadurch werden alle von dir gespeicherten persönlichen Daten (nicht die Nutzendeninhalte) unwiderruflich gelöscht. Möchtest du _eCamp_ wieder nutzen, so musst du dich neu registrieren. Alle Nutzendeninhalte bleiben zwecks Transparenz, Nachvollziehbarkeit und Weiterverwendung in zukünftigen Lagern bestehen, werden jedoch einem anonymen Nutzenden-Konto zugewiesen.
```

Comment:
In diesem Fall würden wir den User-Account deaktivieren, aller persönlichen Properties resetten und den Naen auf irgendeinen Dummy-Wert stellen?

Reply:
Weiss ich noch nicht. Habe aber dafür https://github.com/ecamp/ecamp3/issues/3180 erfasst, wo wir das klären und gleich automatisieren können.

Reply:
Ah nice, thanks